src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const generalCSQuiz = [
+  {
+    Question: "What does NFC stand for?",
+    Answer: "Near-Field Communication",
+    Distractor1: "Narrow Field Communication",
+    Distractor2: "Near-Fragile Communication",
+    Distractor3: "None of the above",
+    Explanation:
+      "NFC stands for near-field communication.",
```

Comment:
it would be helpful to learners to provide a definition of what it is 

```suggestion
      "NFC stands for near-field communication and it is the standard for devices to communicate with each other wirelessly from a very close distance..",
```